.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: emacs-lisp
+sudo: false
+cache: apt
+env:
+  - EVM_EMACS=emacs-25.1-travis
+  - EVM_EMACS=emacs-25.2-travis
+before_install:
+  - git clone https://github.com/rejeep/evm.git /home/travis/.evm
+  - export PATH="/home/travis/.evm/bin:$PATH"
+  - evm config path /tmp
+install:
+  - evm install $EVM_EMACS --use --skip
+  - ln -s "$(pwd)/.emacs.d" ~/.emacs.d
+  - emacs --script .emacs.d/test/lisp/install.el
+  - echo "base-config" > .emacs.d/lisp/user/default-username
+before_script:
+  - emacs --version
+script:
+  - emacs --script ~/.emacs.d/init.el
+  - emacs --script .emacs.d/test/lisp/run-tests.el
```

Comment:
The first one is intended to make sure that Emacs can successfully start without error. The second runs the tests.

The `~/` is just meant to replicate Emacs' behavior by running init.el from the home directory instead of the clone directory.